### PR TITLE
PZ-3913 - Adding info about remainder payment details in guides

### PR DESCRIPTION
**Pull request by MollieRick**

### CODE REVIEW

File: source/orders/integrating-vouchers.rst [hunk 1]
```diff
@@ -58,4 +58,19 @@ Refunds
...
+remainder payment when the consumer partially paid with a voucher. Receiving these details is possible
+in both the :doc:`Payments API </reference/v2/payments-api/get-payment>` and
+:doc:`Orders API </reference/v2/orders-api/get-order>`.
+
+To receive the remainder details in the Payments API, use the ``details.remainderDetails`` include.
+For example: ``GET https://api.mollie.com/v2/payments/tr_xxx?include=details.remainderDetails``. For
+the Orders API you should the ``payments.details.remainderDetails`` embed;
+``GET https://api.mollie.com/v2/orders/ord_xxx?embed=payments.details.remainderDetails``.
+
+Be aware that the ``remainderDetails`` is only available when the payment was not fully paid with a
```
Comment (willemstuursma) [inline source/orders/integrating-vouchers.rst:75]: "...when the payment was partially paid with a gift card and then the remainder was paid with a different payment method."

File: source/guides/gift-cards.rst [hunk 2]
```diff
@@ -110,6 +110,21 @@ Refunds
 You cannot perform any gift card refunds. However, if another payment method was used during the checkout, you can
 refund the part paid with the other payment method (and optionally an additional part).
 
+Getting the details of the remainder payment
+--------------------------------------------
+In some cases it is desirable to receive the details of a payment. This is mostly the case from the
+remainder payment when the consumer partially paid with a gift card. Receiving these details is possible
```
Comment (willemstuursma) [inline source/guides/gift-cards.rst:116]: ...consumer pays with both a gift card and another payment method.

File: source/guides/gift-cards.rst [hunk 3]
```diff
@@ -110,6 +110,21 @@ Refunds
...
+in both the :doc:`Payments API </reference/v2/payments-api/get-payment>` and
+:doc:`Orders API </reference/v2/orders-api/get-order>`.
+
+To receive the remainder details in the Payments API, use the ``details.remainderDetails`` include.
+For example: ``GET https://api.mollie.com/v2/payments/tr_xxx?include=details.remainderDetails``. For
+the Orders API you should the ``payments.details.remainderDetails`` embed;
+``GET https://api.mollie.com/v2/orders/ord_xxx?embed=payments.details.remainderDetails``.
+
+Be aware that the ``remainderDetails`` is only available when the payment was not fully paid with a
+gift card.
```
Comment (ndijkstra) [inline source/guides/gift-cards.rst:126]:
```suggestion
Be aware that the ``remainderDetails`` is only available when the payment is partially paid with a gift card.
```

File: source/guides/gift-cards.rst [hunk 4]
```diff
@@ -110,6 +110,21 @@ Refunds
...
+in both the :doc:`Payments API </reference/v2/payments-api/get-payment>` and
+:doc:`Orders API </reference/v2/orders-api/get-order>`.
+
+To receive the remainder details in the Payments API, use the ``details.remainderDetails`` include.
+For example: ``GET https://api.mollie.com/v2/payments/tr_xxx?include=details.remainderDetails``. For
+the Orders API you should the ``payments.details.remainderDetails`` embed;
+``GET https://api.mollie.com/v2/orders/ord_xxx?embed=payments.details.remainderDetails``.
+
+Be aware that the ``remainderDetails`` is only available when the payment ws partially paid with
+a gift card and then the remainder was paid with a different payment method.
```
Comment (pabloroman) [inline source/guides/gift-cards.rst:126]: What happens if you use `reminderDetails` in a non partially paid payment? Do get an empty object, or no object at all?

File: source/guides/gift-cards.rst [hunk 5]
```diff
@@ -110,6 +110,21 @@ Refunds
...
+remainder payment when the consumer partially paid with a gift card. Receiving these details is possible
+in both the :doc:`Payments API </reference/v2/payments-api/get-payment>` and
+:doc:`Orders API </reference/v2/orders-api/get-order>`.
+
+To receive the remainder details in the Payments API, use the ``details.remainderDetails`` include.
+For example: ``GET https://api.mollie.com/v2/payments/tr_xxx?include=details.remainderDetails``. For
+the Orders API you should the ``payments.details.remainderDetails`` embed;
+``GET https://api.mollie.com/v2/orders/ord_xxx?embed=payments.details.remainderDetails``.
+
+Be aware that the ``remainderDetails`` is only available when the payment ws partially paid with
```
Comment (pabloroman) [inline source/guides/gift-cards.rst:125]: Typo: "the payment **ws** partially paid"

File: source/orders/integrating-vouchers.rst [hunk 6]
```diff
@@ -58,4 +58,19 @@ Refunds
...
+
+Getting the details of the remainder payment
+--------------------------------------------
+In some cases it is desirable to receive the details of a payment. This is mostly the case from the
+remainder payment when the consumer partially paid with a voucher. Receiving these details is possible
+in both the :doc:`Payments API </reference/v2/payments-api/get-payment>` and
+:doc:`Orders API </reference/v2/orders-api/get-order>`.
+
+To receive the remainder details in the Payments API, use the ``details.remainderDetails`` include.
+For example: ``GET https://api.mollie.com/v2/payments/tr_xxx?include=details.remainderDetails``. For
```
Comment (pabloroman) [inline source/orders/integrating-vouchers.rst:71]: Not in the scope of this PR but it makes me a bit sad that the format to include the remainderDetails is different for orders and payments :(

File: source/orders/integrating-vouchers.rst [hunk 7]
```diff
@@ -58,4 +58,19 @@ Refunds
...
 You cannot perform any voucher refunds due to the limitation set by the brand issuers. However,
 if another payment method was used during the checkout, you can refund the part paid with the
-payment method used for the remainer amount.
+payment method used for the remainder amount.
+
+Getting the details of the remainder payment
+--------------------------------------------
+In some cases it is desirable to receive the details of a payment. This is mostly the case from the
+remainder payment when the consumer partially paid with a voucher. Receiving these details is possible
+in both the :doc:`Payments API </reference/v2/payments-api/get-payment>` and
```
Comment (pabloroman) [inline source/orders/integrating-vouchers.rst:67]: Vouchers can't be used in combination with the Payments API, so I think we should remove it here for clarity

File: source/orders/integrating-vouchers.rst [hunk 8]
```diff
@@ -58,4 +58,19 @@ Refunds
 -------
 You cannot perform any voucher refunds due to the limitation set by the brand issuers. However,
 if another payment method was used during the checkout, you can refund the part paid with the
-payment method used for the remainer amount.
+payment method used for the remainder amount.
+
+Getting the details of the remainder payment
```
Comment (pabloroman) [inline source/orders/integrating-vouchers.rst:63]: I would like to rephrase this entire block to something to make it a bit more concise. May I suggest the following:

"You can request additional details (link to https://docs.mollie.com/reference/v2/payments-api/get-payment#payment-method-specific-details) of the multiple payments used to pay a Voucher order using the ``payments.details.remainderDetails`` embed in the Orders API. 

For example ``GET https://api.mollie.com/v2/orders/ord_xxx?embed=payments.details.remainderDetails``

Be aware that the ``remainderDetails`` is only available when the payment was partially paid with
a voucher and then the remainder was paid with a different payment method.

**Review by pabloroman — CHANGES_REQUESTED** [review]

File: source/guides/gift-cards.rst (hunk 4 above)
Comment (MollieRick) [reply source/guides/gift-cards.rst:126]: No object at all

File: source/orders/integrating-vouchers.rst (hunk 6 above)
Comment (MollieRick) [reply source/orders/integrating-vouchers.rst:71]: The one is an include, the order an embed. Thats the biggest (logical) difference

File: source/orders/integrating-vouchers.rst (hunk 7 above)
Comment (MollieRick) [reply source/orders/integrating-vouchers.rst:67]: You can not create a voucher payment with the Payments API, but you **can** get the details of the payment from it.

File: source/orders/integrating-vouchers.rst (hunk 8 above)
Comment (MollieRick) [reply source/orders/integrating-vouchers.rst:63]: Like said before, it is also possible in the Payments API. I think its good to keep the docs as complete as possible.

**Review by pabloroman — APPROVED** [review]

**Pull request merged**